drivers/hmis/app/models/hmis/role.rb

```diff
@@ -236,6 +236,14 @@ def self.permissions_with_descriptions
           'Enrollments',
         ],
       },
+      can_view_limited_enrollment_details: {
+        description: 'Grants access to view limited information about an enrollment, including: entry date, exit date, project name, project type, move-in date, and last bed night date.',
+        administrative: false,
+        access: [:viewable],
+        categories: [
+          'Enrollments',
+        ],
+      },
```

Comment:
I'm starting to feel like this is what we intended the "warehouse" endpoint to handle.  Maybe time to revisit that.

Reply:
Is it? I always thought the 'warehouse endpoint' was meant to expose minimal data from _other_ data sources. But yeah, we should talk about plans for that going forward.

Reply:
As discussed offline, this is probably fine as is, happy to revisit the warehouse endpoint at a future date, especially since this still all falls within the HMIS data source.